core/src/test/scala/unit/kafka/server/ReplicaManagerTest.scala

```diff
@@ -5177,9 +5177,12 @@ class ReplicaManagerTest {
         replicaManager.getPartition(topicPartition) match {
           case HostedPartition.Online(partition) =>
             partition.appendRecordsToFollowerOrFutureReplica(
-              records = MemoryRecords.withRecords(Compression.NONE, 0,
-                new SimpleRecord("first message".getBytes)),
-              isFuture = false
+              records = MemoryRecords.withRecords(
+                Compression.NONE, 0,
+                new SimpleRecord("first message".getBytes)
+              ),
+              isFuture = false,
+              partitionLeaderEpoch = Int.MaxValue
```

Comment:
Should we just use 0 as the leader epoch?

Reply:
Sure. Fixed it. I wanted to make it explicit that this test was "ignoring" the skip higher leader epoch logic.